.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+exclude: '^$'
+fail_fast: false
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 19.10b0
+    hooks:
+    - id: black
+      language_version: python3.6
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+    - id: detect-aws-credentials
+    - id: detect-private-key
+    - id: trailing-whitespace
+    - id: flake8
+      args: ['--ignore=E203, E266, E501, W503, F403, F401']
+#-   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Commenting out mypy pre-commit hook for now. Somehow it would not run in my local deployment, which is weired, b/c I always use the same config.